docs/tutorials/podman_tutorial.md

```diff
@@ -163,7 +163,11 @@ To remove the httpd container:
 $ sudo podman rm --latest
 ```
 You can verify the deletion of the container by running *podman ps -a*.
+
+## Integration Tests
+For more information on how to setup and run the Integrations tests in your environment, checkout the Integration Tests [README](../../test)
```

Comment:
I don't think we need to title-case "Integration" (also you have "Integrations" plural).  For example, see the many lowercase instances [on Wikipedia][1].

[1]: https://en.wikipedia.org/wiki/Integration_testing

Reply:
I've lower cased the first Integerations tests  and have made it singular.  The one before the README.md link I'm leaving  as it matches the page's title.